dotcom-rendering/src/types/front.ts

```diff
@@ -560,3 +560,12 @@ export type TreatType = {
 	 */
 	pageId?: string;
 };
+
+export const NetworkFrontPageIds = [
+	'uk',
+	'us',
+	'au',
+	'international',
+	'europe',
+] as const;
```

Comment:
I'm wondering if we could ensure that these match the editions' ones. Might be too much hard coupling, though…

Reply:
The pageIds could be derived from `edition.ts`

https://github.com/guardian/dotcom-rendering/blob/f339ef0c48e3e0d41d5365c886525c7e34be49d3/dotcom-rendering/src/lib/edition.ts#L8